testing/docker-compose.yml

```diff
@@ -30,33 +30,31 @@ services:
       /usr/bin/mc mb s3/download;
       /usr/bin/mc policy set public s3/download;
       "
-  mq_server:
-    container_name: mq
-    image: ghcr.io/neicnordic/sda-mq:v1.4.1
+  rabbitmq:
+    image: ghcr.io/neicnordic/sensitive-data-archive:${TAG}-rabbitmq
+    container_name: rabbitmq
     environment:
-     - MQ_USER=test
-     - MQ_PASSWORD_HASH=C5ufXbYlww6ZBcEqDUB04YdUptO81s+ozI3Ll5GCHTnv8NAm
-     - MQ_VHOST=test
-     - MQ_VERIFY=verify_none
-     - NOTLS=true
+     - RABBITMQ_DEFAULT_PASS=test
+     - RABBITMQ_DEFAULT_USER=test
+     - MQ_VHOST=sda
     ports:
       - "15672:15672"
-      - "5671:5671"
+      - "5672:5672"
     healthcheck:
-      test: [ "CMD-SHELL", "rabbitmq-diagnostics -q check_running && rabbitmq-diagnostics -q check_local_alarms || exit 1" ]
+      test: ["CMD", "bash", "-c", "rabbitmq-diagnostics -q check_running && rabbitmq-diagnostics -q check_local_alarms"]
```

Comment:
`"CMD-SHELL"` and `"CMD", "bash", "-c"` should behave similarly since you are not doing a bash onliner.

Reply:
Yep, totally agree. This line came in by excessive copy-pasting ;-).